components/serializer.rst

```diff
@@ -162,6 +139,150 @@ needs three parameters:
 2. The name of the class this information will be decoded to
 3. The encoder used to convert that information into an array
 
+Attributes Groups
+-----------------
+
+.. versionadded:: 2.7
+    The support of serialization and deserialization groups was introduced
+    in Symfony 2.7.
+
+Sometimes, you want to serialize different set of attributes from your
+entities. Groups are a handy way to achieve this need.
+
+Assume you have the following object plain-old-PHP object::
+
+    namespace Acme;
+
+    class MyObj
+    {
+        public $foo;
+        public $bar;
+    }
+
+The definition of serialization can be specified using annotations, XML
+or YAML. The :class:`Symfony\\Component\\Serializer\\Mapping\\Factory\\ClassMetadataFactory`
+that will be used by the normalizer must be aware of the format to use.
```

Comment:
For now this is only used for serialization group support. If we add new features in the future, we can update the doc in time.